usaon_benefit_tool/routes/user.py

```diff
@@ -31,7 +31,8 @@ def get(user_id: str):
 @user_bp.route('/<user_id>', methods=['POST'])
 @login_required
 def post(user_id: str):
-    forbid_except_for_roles([RoleName.ADMIN])
+    if user_id != current_user.id and current_user.role_id != RoleName.ADMIN:
```

Comment:
```suggestion
    if user_id != current_user.id:
```

Sorry to request one more change, but we do this check already within `forbid_except_for_roles` :)

https://github.com/Hira-Ijaz/usaon-benefit-tool/blob/cf45b9e48486ac5b7ce587d272133805670248e1/usaon_benefit_tool/util/rbac.py#L21